### PR TITLE
feat: granular thorchain LP deposit and withdraw flags

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box - note, lending, savers and finally swapper will follow this in the stack.

The reason for this order is swaps and LP are the main primitives of THORfi. The naive implementation is to simply be granular per feature, but more research will need to be made on how to simplify things, which will be done in the final swapper PR i.e things could most likely simplified as enabling thorchain interactions (`MsgDeposit` and the like) and pools interactions (all swaps, LP, savers).

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

- contributes to https://github.com/shapeshift/web/issues/6520

> What protocols, transaction types or contract interactions might be affected by this PR?

Low - this should come as no change for THORCHain LP

### Engineering

- THORChain LP pools are shown as halted with the flags off

### Operations

- THORChain LP pools show no regressions against develop

## Screenshots (if applicable)

- LP deposits and withdraws disabled

Screenshot 2024-03-21 at 12 05 49
Screenshot 2024-03-21 at 12 07 50
Screenshot 2024-03-21 at 12 17 27

- LP deposits and withdraws enabled

Screenshot 2024-03-21 at 12 11 55
Screenshot 2024-03-21 at 12 12 07
Screenshot 2024-03-21 at 12 12 16

### CODE REVIEW

**Comment by gomesalexandre** [comment]
* **#6527** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6527?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/pages/ThorChainLP/AvailablePools.tsx [hunk 1]
```diff
@@ -56,8 +57,19 @@ export const AvailablePools = () => {
             swapperName: SwapperName.Thorchain,
           })
 
+          const isThorchainLpDepositEnabled = useFeatureFlag('ThorchainLpDeposit')
+          const isThorchainLpWithdrawEnabled = useFeatureFlag('ThorchainLpWithdraw')
+          const isThorchainLpDepositInteractionDisabled =
```
Comment (gomesalexandre) [inline src/pages/ThorChainLP/AvailablePools.tsx:62]:
```suggestion
          const isThorchainLpInteractionDisabled =
```

Comment (gomesalexandre) [reply src/pages/ThorChainLP/AvailablePools.tsx]: 940a54d5d1

**Review by kaladinlight — APPROVED** [review]
does what it needs to. feels like we could improve with `disabled` vs `halted` vernacular since the pools may not actually be halted and we just chose to disable them for whatever reason.

**Comment by gomesalexandre** [comment]
> does what it needs to. feels like we could improve with `disabled` vs `halted` vernacular since the pools may not actually be halted and we just chose to disable them for whatever reason.

Will bring this to product ! For all intents and purposes it feels like they *would* convey the same meaning but you're absolutely right we may want to disable some THOR feature in case there's e.g a critical issue on our/THOR side while the pool isn't explicitly halted, and we may want to word this differently

**Pull request merged**